middlewares/validators/tasks.js

```diff
@@ -262,10 +263,21 @@ const getUsersValidator = async (req, res, next) => {
     res.boom.badRequest(error.details[0].message);
   }
 };
+
+const filterOrphanTasksValidator = async (req, res, next) => {
+  try {
+    await validateMillisecondsTimestamp(req.body, "lastOrphanTasksFilteration");
```

Comment:
- why are we doing this?

Reply:
to validate if we are receiving timestamp, just following consistency 🧷